### PR TITLE
Nuke submission own terminal

### PR DESCRIPTION
**Pull request by pKrime**
As mentioned in issue #4, rrSubmitter will change colors and write messages to nuke's terminal.

An optional flag can be added to start rrSubmitter in its own terminal (at present, windows only).

cons: I have not yet found a way to keep rrSubmitter's terminal open, so some debug messages might be lost.

### CODE REVIEW

File: _submitplugins/rrSubmit_Nuke_5.py [hunk 1]
```diff
@@ -246,6 +246,11 @@ def submitJobsToRR(jobList, submitOptions, nogui=False):
         commandline=getRRSubmitterconsolePath() + "  \"" + tmpFile.name + "\"" + ' -Wait'
     else:
         commandline= getRRSubmitterPath()+"  \""+tmpFile.name+"\""
+
+    if own_terminal:
+        if sys.platform.lower().startswith("win"):
+            commandline = 'START CMD /C "{0}"'.format(commandline)
```
Comment (JenusL) [inline _submitplugins/rrSubmit_Nuke_5.py]: Maybe the problem here is that you have quotes around `{0}` causing the actual command to have double, or inverted quotes.
Try without quotes but with a title:
`commandline = 'START "rrSubmitter" CMD /C {0}'.format(commandline)`

Comment (pKrime) [reply _submitplugins/rrSubmit_Nuke_5.py]: I had misunderstood how START works. Please check my latest commit

Comment (JenusL) [reply _submitplugins/rrSubmit_Nuke_5.py]: Looks good to me if it works for you.

**Comment by JenusL** [comment]
Looks good now... but could you add the kwarg to `rrSubmit_Nuke` as well?
Line 619: `def rrSubmit_Nuke(own_terminal=False):`
and line 636: `submitJobsToRR(jobList,submitOptions, own_terminal=own_terminal)`
It doesn't matter if it's default to True or False.

**Comment by pKrime** [comment]
Ok, in that case I will default _own_terminal_ to False: since the default behaviour stays the same, I can merge immediately

**Pull request merged**